SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Reporting a Vulnerability
+We take security bugs in SQLitebrowser and related project seriously.  
```

Comment:
We should probably say "DB Browser for SQLite" instead of "SQLitebrowser" here, as that's the official name we changed stuff to years ago.  Even though we've kept the original domain name and other stuff. :smile:

Reply:
Cool, I fixed it. :)